tensorflow/contrib/tensorrt/__init__.py

```diff
@@ -18,6 +18,16 @@
 from __future__ import division
 from __future__ import print_function
 
-# pylint: disable=unused-import,wildcard-import
-from tensorflow.contrib.tensorrt.python import *
-# pylint: enable=unused-import,wildcard-import
+# pylint: disable=unused-import,wildcard-import,g-import-not-at-top
+try:
+  from tensorflow.contrib.tensorrt.python import *
+except Exception as e:
```

Comment:
We should not be using general `Exception` types.
This will throw an `ImportError` so we should use that instead.
Could you send a followup fix for that?

Reply:
Just to be clear here.
Are you suggesting to 
a. catch general ```Exception``` and throw an ```ImportError```
Or
b. catch only ```ImportError``` and throw it,

Reply:
I think it'll actually throw a `tensorflow.python.framework.errors_impl.NotFoundError`.

Reply:
I notice that it throws 
```NotFoundError: libnvinfer.so.4: cannot open shared object file: No such file or directory``` 

I could catch ```Exception``` and throw an ```ImportError``` with the original text attached after our warning. so that any error happens our message gets printed.

```
ImportError: **** Failed to initialize TensorRT. This is either because the TensorRT installation path is not in LD_LIBRARY_PATH, or because you do not have it installed. If not installed, please go to https://developer.nvidia.com/tensorrt to download and install TensorRT ****
libnvinfer.so.4: cannot open shared object file: No such file or directory
```

Reply:
Don't we only want to catch the exception type we expect?

Reply:
@gunan @yifeif Then how about something like:
```
try:
  from tensorflow.contrib.tensorrt.python import *
except NotFoundError as e:
  if e contains 'libnv.*: cannot open shared object file':
    no_trt_message = ...
    raise ImportError(no_trt_message, e)
  else:
    raise e
```

Reply:
I think we can just change what we have now from `except Exception as e:` to `except NotFoundError as e:`

Reply:
I am really puzzled, I cannot see any code path that throws the NotFoundError.
I think NotFoundError gets thrown when we are executing a graph.

Is there a way to catch this during tensorrt import/access time?